src/sentry/user_feedback/blueprints/api.md

```diff
@@ -0,0 +1,235 @@
+| url           | string           | The page the user-feedback was triggered on.        |
+| user          | object           | The authorized user's information.                  |
+
+- Response 200
+
+  ```json
+  {
+    "data": [
+      {
+        "dist": "abc123",
+        "email": "colton.allen@sentry.io",
+        "environment": "production",
+        "error_ids": ["4f0e18332d11431c9b97d924edecb76e"],
+        "feedback_text": "I really like this user-feedback feature!",
+        "id": "1ffe0775ac0f4417aed9de36d9f6f8dc",
+        "locale": {
+          "lang": "en",
+          "timezone": "UTC+1"
+        },
+        "platform": "javascript",
+        "release": "version@1.3",
+        "replay_id": "ec3b4dc8b79f417596f7a1aa4fcca5d2",
+        "request": {
+          "headers": {
+            "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.0.0 Safari/537.36"
+          }
+        },
+        "sdk": {
+          "name": "sentry.javascript.react",
+          "version": "6.18.1"
+        },
+        "status": "unresolved",
+        "tags": {
+          "key": "value"
+        },
+        "timestamp": "2023-08-31T14:10:34.954048",
+        "trace_ids": ["fbb42a39a61e4aa6969b4fecbfbbc114"],
+        "url": "https://docs.sentry.io/platforms/javascript/",
+        "user": {
+          "id": "123",
+          "username": "username",
+          "email": "username@example.com",
+          "ip_address": "127.0.0.1"
+        }
+      }
+    ]
+  }
+  ```
+
+### Create User Feedback [POST]
+
+Create a new user-feedback item.
+
+- Request
+
+  ```json
+  {
+    "data": {
+      "dist": "abc123",
+      "email": "colton.allen@sentry.io",
+      "environment": "production",
+      "error_ids": ["4f0e18332d11431c9b97d924edecb76e"],
+      "feedback_text": "I really like this user-feedback feature!",
+      "id": "1ffe0775ac0f4417aed9de36d9f6f8dc",
+      "platform": "javascript",
+      "release": "version@1.3",
+      "replay_id": "ec3b4dc8b79f417596f7a1aa4fcca5d2",
+      "request": {
+        "headers": {
+          "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.0.0 Safari/537.36"
+        }
+      },
+      "sdk": {
+        "name": "sentry.javascript.react",
+        "version": "6.18.1"
+      },
+      "status": "unresolved",
+      "tags": {
+        "key": "value"
+      },
+      "timestamp": "2023-08-31T14:10:34.954048",
+      "trace_ids": ["fbb42a39a61e4aa6969b4fecbfbbc114"],
+      "url": "https://docs.sentry.io/platforms/javascript/",
+      "user": {
+        "id": "123",
+        "username": "username",
+        "email": "username@example.com",
+        "ip_address": "127.0.0.1"
+      }
+    }
+  }
+  ```
+
+## User Feedback [/projects/<organization_slug>/<project_slug>/user-feedback/<user_feedback_id>/]
+
+- Model
+
+  - Body
+
+    ```json
+    {
+      "data": {
```

Comment:
we should add a top level "user_feedback" object here for user feedback specific items

Reply:
feedback_text and feedback_email should go in here.

Reply:
status as well.